src/main/scala/lms/transformation/mpi_nccl_transformation/distributedTensor2MPINCCL.scala

```diff
@@ -27,6 +27,9 @@ class DistributeTensor2MPI_NCCLAnalysis extends Traverser {
         case Node(s, op, _, _) if op.startsWith("tensor_dot") =>
             hasCublas = true
             super.traverse(n)
+        case Node(s, op, _, _) if op.startsWith("cudnn") =>
```

Comment:
I would prefer to keep some naming conventions when designing node ops names
Here I would prefer that the ops (or at least those with single output) has names starting with `tensor_xxx`

Reply:
How about `tensor_cudnn`?